pipeline/parse/parsers/Telegram.d.ts

```diff
@@ -11,8 +11,8 @@ type TelegramChannelType =
 
 // some available keys: id, type, date, actor, actor_id, action, title, text, from, from_id, reply_to_message_id, file, thumbnail, media_type, sticker_emoji, width, height, photo, via_bot, mime_type, duration_seconds, edited, inviter, forwarded_from, message_id, members, performer, game_title, game_description, game_link, game_message_id, score, place_name, address, location_information, poll, saved_from, author, duration
```

Comment:
I have to wonder why date_unixtime and edited_unixtime aren't listed here. I understand it says "some", but I can find people experiencing similar issues back in mid 2021 and this comment was added in 2022. Didn't want to add it in case there was a reason for this- I have never used Telegram so it could be something obvious I just don't know about haha

Reply:
Whoops commented below